### ISSUE TITLE
tokenizer can not return offset_map

### ISSUE THREAD

**Issue by WallE-Chang** [NormXU/ERNIE-Layout-Pytorch#4]
Hi, I read your tokenizer code which is subclass of PretrainedTokenizer. But PretrainedTokenizer of paddlenlp is more similar to PretrainedTokenizerFast of transformers, which means tokenizer can return offset. The code as following 
```python
content_encoded_inputs = tokenizer(
  text=[prompt],
  text_pair=[this_text_line],
  max_seq_len=max_seq_len,
  return_dict=False,
  return_offsets_mapping=True,
)

```

**Comment by WallE-Chang** [comment]
I have reviewed the code of  PreTrainedTokenizerBase in t[okenizer_utils.py](https://github.com/NormXU/ERNIE-Layout-Pytorch/blob/main/networks/tokenizer_utils.py). However, the  return_offsets_mapping dosen't return offset 
<img width="792" alt="image" src="https://user-images.githubusercontent.com/22662737/211281303-e144f1cd-e3cc-4475-882b-00c1ae040429.png">

**Comment by WallE-Chang** [comment]
I find the reason. The prepare_for_model funtion of PreTrainedTokenizerBase  in  paddlenlp and  transfromers is different.  prepare_for_model in paddlenlp  have return offset , but transfromer dosen't .
This is the code how paddlenlp handling offser . https://github.com/PaddlePaddle/PaddleNLP/blob/develop/paddlenlp/transformers/tokenizer_utils_base.py#L2764

**Comment by NormXU** [comment]
fix the bug [commit-d827f5cac3d6973738e233bf0381e4b374dd6c3f](https://github.com/NormXU/ERNIE-Layout-Pytorch/commit/d827f5cac3d6973738e233bf0381e4b374dd6c3f)
